### ISSUE TITLE
Can OBJ file be used in PoissionRecon

### ISSUE THREAD

**Issue by JacobYeah** [mkazhdan/PoissonRecon#213]
I have tried to reconstruct OBJ 3D file using PoissionRecon and failed. Is there any solution?

**Comment by mkazhdan** [comment]
You can try using MeshLab to convert to PLY format.

**Comment by JacobYeah** [comment]
Thank you very much! Another question, how to run the PoissonRecon using Debug mode, I have tried but it has error that cannot open including file"ZLIB/zlib.h", but I have configured it. So confusing

**Comment by mkazhdan** [comment]
You need to copy the include directories from the Release version. You'll also need to add to link to the appropriate libraries. (Again, you can copy from the Release settings.)

**Comment by JacobYeah** [comment]
Thank you. But what do you mean "copy the include directories from Release version"?

**Comment by mkazhdan** [comment]
In the project properties you can specify where the compiler should look for header files. You need to copy the directories specified for the release version to the properties for the debug version. (Same goes for the libraries you want the linker to link against.)

**Comment by JacobYeah** [comment]
Exactly, I have done that. But still remind me: no such file or directory
